modules/Microsoft.Compute/virtualMachines/readme.md

```diff
@@ -13,105 +13,104 @@ This module deploys one Virtual Machine with one or multiple nics and optionally
 
 ## Resource Types
 
-| Resource Type | API Version |
-| :-- | :-- |
-| `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks) |
-| `Microsoft.Authorization/roleAssignments` | [2022-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2022-04-01/roleAssignments) |
-| `Microsoft.Automanage/configurationProfileAssignments` | [2021-04-30-preview](https://docs.microsoft.com/en-us/azure/templates) |
-| `Microsoft.Compute/virtualMachines` | [2021-07-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Compute/2021-07-01/virtualMachines) |
-| `Microsoft.Compute/virtualMachines/extensions` | [2021-07-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Compute/2021-07-01/virtualMachines/extensions) |
-| `Microsoft.Insights/diagnosticSettings` | [2021-05-01-preview](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Insights/2021-05-01-preview/diagnosticSettings) |
-| `Microsoft.Network/networkInterfaces` | [2021-08-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/2021-08-01/networkInterfaces) |
-| `Microsoft.Network/publicIPAddresses` | [2021-08-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Network/2021-08-01/publicIPAddresses) |
+| Resource Type                                                                         | API Version                                                                                                                                                   |
```

Comment:
Could you regenerate this file using the [ReadMe generator](https://github.com/Azure/ResourceModules/wiki/Contribution%20guide%20-%20Generate%20module%20Readme)? Seems there are some extensions to VSCode which is formatting your tables, which we have disabled with the .vscode settings.

Reply:
Hi @MariusStorhaug. Thanks. This was not intentional. It is caused by Windows converting LF to CRLF at the end of lines. I have fixed this in the latest commit. Please check and let me know if you're happy.

Reply:
Thank you for the work and contribution! :D Fixed the last thing, just a comma.